week-8-practise/1-code-refactoring.js

```diff
@@ -8,28 +8,28 @@
  * There may be both syntax and logical errors present.
  */
 
-var reversed;
-let capitilised;
 
-// Function that reverses and capitlises text
-exports function reverseAndCaptilise(text) 
-  var capitlised = text.toUpper()
-  reversed = capitilised.split('').reverse().join('') 
-
-  return reversed
-  }
+// this function handle the split the reverse ad the join method
+export function reverseAndCaptilise(text) {
+  return text
+    .split("")
+    .reverse()
+    .join("")
+    .toUpperCase();
+}
 
-// Function that returns text with every other letter capitlised
+//this function will transform all text to upperCase,
 export function everyOtherLetterCapitilised(text) {
```

Comment:
What do you think about the naming of the function?

Reply:
@sansaid This is the toughest part, has decided to just rename the function upperCase, since that is the main purpose of the  function, will be pushing the update shortly